trunk/App.config

```diff
@@ -229,14 +232,20 @@
       <setting name="ParcelArea" serializeAs="String">
         <value>512</value>
       </setting>
+      <setting name="ParcelID" serializeAs="String">
+        <value>00000000-0000-0000-0000-000000000000</value>
+      </setting>
+      <setting name="ParcelSeeAvatars" serializeAs="String">
+        <value>0</value>
+      </setting>
       <setting name="VersionControl" serializeAs="String">
         <value>True</value>
       </setting>
       <setting name="VersionControlSVN" serializeAs="String">
         <value>True</value>
       </setting>
       <setting name="OutputFormat" serializeAs="String">
-        <value>UTF-8</value>

```

Comment:
Curious about this change. UTF-8 is the official name for the format according to the Unicode Consortium, so why change it?

Reply:
Still wondering about this ;-)